itomate/itomate.py

```diff
@@ -181,11 +181,18 @@ async def activate(connection):
             continue
 
         # Set root path if it exists
-        if root_path:
-            for pane in tab_panes:
-                commands = pane.get('commands') or []
+        for pane in tab_panes:
+            commands = pane.get('commands') or []
+            if root_path:
                 commands.insert(0, f"cd {root_path}")
-                pane['commands'] = commands
+                for command in commands:
```

Comment:
Move this loop out of the root_path conditional and we will be good to go

Reply:
Up to this point, wouldn't the `commands` array be empty if the `root_path` is falsey?

Reply:
Not necessarily - I misread the code here.

Reply:
root_path is optional